src/main/scala/ie/zalando/fabric/gateway/service/IngressDerivationChain.scala

```diff
@@ -568,6 +575,13 @@ class IngressDerivationChain(stackSetOperations: StackSetOperations, versionedHo
           .mkString("{", ",", "}"))
   }
 
+  def filterAllowedAnnotations(meta: GatewayMeta, allowedAnnotationKeys: Set[String]): Map[String, String] = {
+    val allowedAnnotations = meta.annotations.filterKeys(allowedAnnotationKeys.contains)
+    log.debug(s"Annotations filter from ${meta.annotations} and passed through as $allowedAnnotations")
```

Comment:
Maybe we should change this to log a warn if annotations have been filtered out

Reply:
Annotations will always be filtered out. The likes of deployment-time etc. There's extra annotations that we receive from what people may actually add to their resource